tests/src/ondevice_tests/image_align_node_test.cpp

```
@@ -1,92 +1,227 @@
+constexpr unsigned kDepthHeight = 48;
+constexpr float kFocalLength = 100.0f;
+constexpr float kPrincipalPointX = static_cast<float>(kDepthWidth) / 2.0f;
+constexpr float kPrincipalPointY = static_cast<float>(kDepthHeight) / 2.0f;
 
-    auto alignedQueue = align->outputAligned.createOutputQueue();
-    auto alignToQueue = rgbOut->createOutputQueue();
-    p.start();
+std::array<std::array<float, 3>, 3> makeIntrinsics() {
+    return {{{kFocalLength, 0.0f, kPrincipalPointX}, {0.0f, kFocalLength, kPrincipalPointY}, {0.0f, 0.0f, 1.0f}}};
+}
 
```

Cppcheck (2.20.0)

[error] There is an unknown macro here somewhere. Configuration is required. If DEPTHAI_NLOHMANN_DEFINE_TYPE_INTRUSIVE is a macro then please configure it.

(unknownMacro)